types/cache-interceptor.d.ts

```diff
@@ -56,6 +57,11 @@ declare namespace CacheHandler {
    * Underlying storage provider for cached responses
    */
   export interface CacheStore {
+    /**
+     * Whether or not the cache is full and can not store any more responses
+     */
+    isFull?: Readonly<boolean>
```

Comment:
I still don't see the point of this. The cache is never full. If it's full then the older values should be removed instead of not caching the most recent value.